### PR TITLE
Add badges with the release info to the README.md

### PR DESCRIPTION
**Pull request by EgorPopelyaev**
This PR adds the badges to the readme to show the info about the current stable release version and the version and date of the upcoming stable release. The idea here to give a possibility for people from the outside to have an overview about the current and upcoming release, until we have a proper public release calendar.

Screenshot 2024-08-29 at 14 47 57

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,8 @@ curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/paritytec
 
 ## 🚀 Releases
 
+![Current Stable Release](https://img.shields.io/badge/Current%20Stable%20Release-polkadot_2407_1-green)&nbsp;&nbsp;![Next Stable Release (polkadot-2409)](https://img.shields.io/badge/Next%20Stable%20Release%20%28polkadot_2409%29-2024/09/25-orange)
```
Comment (bkchr) [inline README.md:41]: And you want to update them manually?

Comment (EgorPopelyaev) [reply README.md:41]: At the beginning yes, as part of the version bumps backport routine, when release is done. Later we could, probably, automate it using the [release-registry](https://github.com/paritytech/release-registry) as a data source

**Review by bkchr — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (bkchr) [reply README.md:41]: Okay, I hope you then ensure that this happens :D

**Review by skunert — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ggwpez) [reply README.md:41]: Actually, we can do this already now. Please use these URLs instead:  

`https://raw.githubusercontent.com/paritytech/release-registry/main/badges/polkadot-sdk-latest.svg`
`https://raw.githubusercontent.com/paritytech/release-registry/main/badges/polkadot-sdk-next.svg`

(test if they render here)
  
a
b

Comment (EgorPopelyaev) [reply README.md]: I've changed the URL's, looks good!

**Comment by kianenigma** [comment]
I mentioned to @ggwpez elsewhere that the current information under the `Release` heading our README is probably by now not relevant, and be replaced with a link to the release registery? would be nice to do it as a part of this. 

btw, the registery is super 🤌, love it.

**Comment by EgorPopelyaev** [comment]
> I mentioned to @ggwpez elsewhere that the current information under the `Release` heading our README is probably by now not relevant, and be replaced with a link to the release registery? would be nice to do it as a part of this.
> 
> btw, the registery is super 🤌, love it.

Do you mean the text part itself?
Screenshot 2024-09-04 at 12 54 58

**Comment by ggwpez** [comment]
Screenshot 2024-09-04 at 13 43 03

Updated it. Looks like this now.

**Pull request merged**